cobbler/modules/managers/import_signatures.py

```diff
@@ -84,11 +87,15 @@ def import_walker(top, func, arg):
             import_walker(name, func, arg)
 
 
-class ImportSignatureManager(object):
+class ImportSignatureManager:
+    """
+    This class contains most of the magic behind the ``cobbler import`` command. It uses the signatures as a data source
+    and then automatically adds the detected distro to Cobbler.
+    """
 
-    def __init__(self, collection_mgr, logger):
+    def __init__(self, collection_mgr, logger: clogger.Logger):
         """
-        Constructor
+        Main constructor for out class.
```

Comment:
What does "out class" exactly mean?

Reply:
Typo.  `r` and `t` are neighboring characters on my layout.